relay-server/src/actors/store.rs

```diff
@@ -78,35 +82,65 @@ pub struct StoreEnvelope {
     pub scoping: Scoping,
 }
 
+/// TODO
+#[derive(Clone, Debug)]
+pub struct StoreMetrics {
+    pub buckets: Vec<Bucket>,
+    pub scoping: Scoping,
+    pub retention: u16,
+    pub mode: ExtractionMode,
```

Comment:
The store actor now logs outcomes internally for metric buckets. I think this is generally desirable, as this means we do not have to await the message sent in. Ideally, I'd like to do the same for envelopes.

However, this means we have to add things like the `ExtractionMode` now to the message so that it can accurately track how much was dropped. 

We could also just send SourceQuantities in, but then we do not know the precise number of transactions or profiles dropped if only some metrics don't make it in.